packages/bsky/src/api/com/atproto/admin/reverseModerationAction.ts

```diff
@@ -77,12 +77,27 @@ export default function (server: Server, ctx: AppContext) {
           { create, negate },
         )
 
-        return result
+        return { result, restored }
       })
 
+      if (restored) {
+        const { did, subjects } = restored
+        const agent = await ctx.pdsAdminAgent(did)
+        await Promise.all(
+          subjects.map((subject) =>
+            agent.api.com.atproto.admin.updateSubjectStatus({
+              subject,
+              takedown: {
+                applied: false,
+              },
+            }),
```

Comment:
We have a helper `retryHttp()` for retrying http requests—might recommend plopping it in here and anywhere else we want to ensure a status gets pushed out to the PDS.

Reply:
yeah good call 👍